projects.json

```diff
@@ -3307,6 +3314,12 @@
       {
         "action": "BuildSwiftPackage",
         "configuration": "release",
+        "xfail": [
+          {
+            "issue": "rdar://83452858",
+            "compatibility": ["5.0"],
```

Comment:
I think this is the issue here, json doesn't allow `,` without subsequent elements.

Reply:
😭 I'm used to having a more lenient json parser.  Thanks